### ISSUE TITLE
Add link to IRC logs and webirc client on the community page

### ISSUE THREAD

**Issue by nixpanic** [gluster/glusterweb#168]
The community page on https://www.gluster.org/community/ could use some enhancements for the IRC logs:

- https://botbot.me/freenode/gluster/
- https://botbot.me/freenode/gluster-dev/
- https://botbot.me/freenode/gluster-meeting/

There is also https://webchat.freenode.net/ that can be used for users new to IRC. The channel list is modifiable by passing the channel as URL parameter like https://webchat.freenode.net/?channels=#gluster

**Comment by sammcj** [comment]
So pretty :)

**Comment by amye** [comment]
This is part of the general content overhaul, adding to that ticket.

**Comment by amye** [comment]
@amarts, this should probably also be on your radar.

**Comment by Legogris** [comment]
The botbot.me logs are no longer working/active since 2018. Any replacement?

**Comment by sankarshanmukhopadhyay** [comment]
I do not think we stood up a bot based logging mechanism once the regulations around data processing, data retention and data erasure came about. Keeping in mind those, it is unlikely that we'll be setting up another one formally.
